src/views/Swap.vue

```diff
@@ -227,8 +227,12 @@ export default class Swap extends Mixins(TranslationMixin, LoadingMixin, NumberF
     return this.isZeroFromAmount && this.isZeroToAmount
   }
 
+  get isXorOutputSwap (): boolean {
+    return this.tokenTo && this.tokenTo?.address === KnownAssets.get(KnownSymbols.XOR)?.address
```

Comment:
I think checking ```this.tokenTo && ....``` is unnecessary

I think using ```this.tokenTo?.address``` is sufficient

Reply:
Thank you for comment. Good point)